edx_name_affirmation/tests/test_signals.py

```diff
@@ -0,0 +1,58 @@
+"""
+Tests for Name Affirmation signals
+"""
+
+import ddt
+
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from edx_name_affirmation.signals import _listen_for_lms_retire_verified_names
+
+from edx_name_affirmation.models import VerifiedName
+
+User = get_user_model()
+
+
+@ddt.ddt
+class RetirementSignalVerifiedNamesTest(TestCase):
+    """
+    Tests for the LMS User Retirement signal for Verified Names
+    """
+
+    def setUp(self):
+        self.user = User(username='tester', email='tester@test.com')
+        self.user.save()
+        self.name = 'Jonathan Smith'
+        self.profile_name = 'Jon Smith'
+        self.idv_attempt_id = 1111111
+        self.proctoring_attempt_id = 2222222
+        self.verified_name_obj = VerifiedName.objects.create(
+            user=self.user,
+            verified_name=self.name,
+            profile_name=self.profile_name,
+            verification_attempt_id=self.idv_attempt_id
+        )
+
+        self.other_user = User(username='other_tester', email='other_tester@test.com')
+        self.other_user.save()
+        self.other_name = 'Jonathan Other'
+        self.other_profile_name = 'Jon Other'
+        self.other_idv_attempt_id = 1111112
+        self.other_proctoring_attempt_id = 2222223
+        self.verified_name_obj = VerifiedName.objects.create(
+            user=self.other_user,
+            verified_name=self.other_name,
+            profile_name=self.other_profile_name,
+            verification_attempt_id=self.other_idv_attempt_id
+        )
+
+    def test_retirement_signal(self):
```

Comment:
Guessing this is good test coverage wise?

Reply:
I think so. You're covering that a user is properly retired and that other users are not retired. The only other thing I could think of is creating 2+ verified names for the retired user to test that all `VerifiedNames` associated with the user are removed and not just one (i.e. the most recent or something).

Reply:
That's a great idea! I've added this test.